### PR TITLE
Sequences max 10k cells

### PR DESCRIPTION
**Pull request by kjetilh33**
Fix batching of sequence headers to respect the limit of max 10k cells per batch

### CODE REVIEW

File: src/main/java/com/cognite/client/Sequences.java [hunk 1]
```diff
@@ -195,6 +193,36 @@ public Aggregate aggregate(Request requestParameters) throws Exception {
...
+            }
+            // Add item to batch and bump the cell counter.
+            currentBatch.add(sequenceMetadata);
+            batchCellsCounter += sequenceMetadata.getColumnsCount();
+        }
+        if (currentBatch.size() > 0) {
+            // Add all remaining items
+            batchList.add(currentBatch);
+        }
+        return batchList;
```
Comment (pavelzubarev) [inline src/main/java/com/cognite/client/Sequences.java:223]: Could be simplified with `Lists.partition` call. ref: https://www.baeldung.com/java-list-split

Comment (pavelzubarev) [reply src/main/java/com/cognite/client/Sequences.java:223]: Basically first you batch columns, then, you batch requests

Comment (kjetilh33) [reply src/main/java/com/cognite/client/Sequences.java:223]: Yes, good catch on the partition/list part. I was debating a bit with myself on using Guava or if we can plan on removing the overall dependency on Guava. I am all fine with switching to Guava--but maybe as a separate change/PR? There are multiple places in the code base where we can do this refactor. 

On the batching: We need to watch two limits, no of items and no of columns. Whichever limit we hit first must trigger a batch boundary.

Comment (pavelzubarev) [reply src/main/java/com/cognite/client/Sequences.java:223]: I think Beam SDK has it already, so it doesn't make a difference. I understood the logic behind it, but less code is more, IMO :)

**Review by pavelzubarev — APPROVED** [review]

**Pull request merged**